docs/AboutUs.md

```diff
@@ -2,51 +2,38 @@
 
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
-## Project Team
+## Project Team ##
 
-#### [Damith C. Rajapakse](http://www.comp.nus.edu.sg/~damithch) <br>
-<img src="images/DamithRajapakse.jpg" width="150"><br>
-**Role**: Project Advisor
+#### [Sean Zhang](http://github.com/puzzledsean) ####
+<img src="images/SeanZhang.jpg" width="150"><br>
+
+Role: Team Lead
```

Comment:
you need to indicate who is in charge of what component in the project, like UI, Logic etc.,